components/dashboard/frontend/src/app/stores/ExplorerStore.tsx

```diff
@@ -121,35 +121,35 @@ class PendingMana {
     timestamp: number;
 }
 
-class Conflict {
+class Spend {
     id: string;
     parents: Array<string>;
-    conflictIDs: Array<string>;
+    spendIDs: Array<string>;
     confirmationState: number;
 }
 
-class ConflictChildren {
-    conflictID: string;
-    childConflicts: Array<ConflictChild>
+class SpendChildren {
+    spendID: string;
+    childSpends: Array<SpendChild>
 }
 
-class ConflictChild {
-    conflictID: string;
+class SpendChild {
+    spendID: string;
     type: string;
 }
 
-class ConflictConflict {
+class SpendSpend {
     outputID: OutputID;
-    conflictIDs: Array<string>;
+    spendIDs: Array<string>;
 }
 
-class ConflictConflicts {
-    conflictID: string;
-    conflicts: Array<ConflictConflict>
+class SpendSpends {
+    spendID: string;
+    spends: Array<SpendSpend>
 }
 
-class ConflictVoters {
-    conflictID: string;
+class SpendVoters {

```

Comment:
I wonder if this here actually isn't about conflicts?

Reply:
I removed those conflict stores and classes instead, bc it's not even implemented and enabled.